File: backend/NoPressure.Common/Models/Team/TeamInfo.cs
using System.Drawing;
using NoPressure.Common.DTO;
using NoPressure.Common.Enums;
using NoPressure.Common.Models.Tag;

namespace NoPressure.Common.Models.Team
{
    public class TeamInfo
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime Date { get; set; }
        public string Color { get; set; }
        public TeamAccess AddingActivities { get; set; } = TeamAccess.Deny;
        public List<TeamTag>? Tags { get; set; }
    }
}

File: frontend/src/app/models/team/team-info.ts
import { TeamAccess } from "../enums/TeamAccess";
import { TeamTag } from "../tag/team-tag";

export interface TeamInfo {
    id: number;
    name: string;
    date: Date;
    color: string;
    addingActivities: TeamAccess;
    tags: TeamTag[];
}